### DISCUSSION TITLE
Just an idea: Login system via university portal

### DISCUSSION THREAD

**Discussion by GLARKI** [community/community#140331] [GitHub Education]
### Hi everyone,

For example, for Zoom or Lime Survey, we can log in via a portal; it refers us to our university login, and after login in, it confirms the affiliation. Wouldn't such a process be an idea?
- I submitted a photo of my student card, but it's not accepted because it's not dated.
I submitted my agreement for extra courses for my doctoral study, but it was not accepted because it contained keywords that made it look like an acceptance letter.
- I've uploaded proof of payment for this semester -> the proof was from our student portal with the student portal logo on it, also not suitable for some reason.
I have the photo of my contract I cannot get sharp over my camera, I guess there are too many pages.

What should I do? There must be a much easier way, maybe via Orcid or some other portal.

**Comment by Thevandieg** [comment]
Use University-Issued Email: If possible, make sure you’re applying with an email address issued by your university (e.g., .edu or equivalent). GitHub often accepts this as automatic verification.

Accepted Forms of Documentation:

Ensure that your student ID card has an expiration date or is clearly marked as valid for the current academic year.
If you're submitting an enrollment agreement or proof of payment, double-check that it explicitly states you’re enrolled for the current term.
Clear Documentation: For blurry or unclear documents (like your contract), consider using a scanning app on your phone, which might produce clearer, more readable documents. Many apps have features that optimize text quality for such cases.

Other Suggestions: You mentioned portals like Orcid. While GitHub doesn't currently integrate with these systems, you could try reaching out to GitHub support and suggesting this as a potential verification method. For now, the documentation method is the primary option.

If you're still having trouble after following these tips, I would recommend contacting GitHub support directly to explain your situation and ask for guidance on what specific documents they will accept.

I hope this helps!

**Comment by GLARKI** [comment]
Thank you very much for the advice.

Op wo 2 okt 2024 om 05:36 schreef Ivan Mendoza ***@***.***>:
